src/content/factory/widget-sourcemaps-manager.factory.ts

```diff
@@ -0,0 +1,24 @@
+import * as path from "path";
+import { FatalError, logger } from "../../util/logger";
+import { WidgetSourcemapsManager } from "../manager/widget-sourcemaps.manager";
+import { WidgetManagerFactory } from "./widget-manager.factory";
+
+export class WidgetSourcemapsManagerFactory {
+    private widgetManagerFactory = new WidgetManagerFactory();
+
+    public createManager(tenantIndependent: boolean = false, userSpecific: boolean = false): WidgetSourcemapsManager {
```

Comment:
`tenantIndependent` and `userSpecific` seem not to be used at all?

Reply:
Yeah I will remove them